sql/field.cc

```diff
@@ -2196,7 +2196,7 @@ Field *Field::make_new_field(MEM_ROOT *root, TABLE *new_table,
   tmp->flags&= (NOT_NULL_FLAG | BLOB_FLAG | UNSIGNED_FLAG |
                 ZEROFILL_FLAG | BINARY_FLAG | ENUM_FLAG | SET_FLAG);
   tmp->reset_fields();
-  tmp->field_visibility= NOT_INVISIBLE;
+  tmp->invisible= VISIBLE;
```

Comment:
I'd argue `invisible= VISIBLE` is great. :)
Probably `visibility` would've been better here.